### DISCUSSION TITLE
How to Install BthPS3 to use it with DSHIDMINI ?

### DISCUSSION THREAD

**Discussion by l3gi0n0fh3ll** [nefarius/DsHidMini#140] [Q&A] [answered]
Is installing **BthPS3Setup_x64.msi** enough ? Or should I install **Shibari.zip** , **FireShock.zip** and **ViGEm Bus Driver** as well ?

**Comment by Kanuan** [comment, marked as answer]
[how to install dshidmini](https://vigem.org/projects/DsHidMini/How-to-Install/)

[Frequently asked questions about dshidmini](https://vigem.org/projects/DsHidMini/FAQ/)

**Comment by l3gi0n0fh3ll** [comment]
The links you posted do not answer my questions....

**Reply by nefarius** [reply]
It's literally all outlined in the links provided, take your time and read them instead of demanding spoonfeeding.

**Comment by l3gi0n0fh3ll** [comment]
Alright, I figured it out...
